homeassistant/components/bosch_shc/entity.py

```diff
@@ -3,13 +3,16 @@
 
 from boschshcpy import SHCDevice, SHCIntrusionSystem
 
+from homeassistant.core import HomeAssistant
 from homeassistant.helpers.device_registry import async_get as get_dev_reg
 from homeassistant.helpers.entity import DeviceInfo, Entity
 
 from .const import DOMAIN
 
 
-async def async_remove_devices(hass, entity, entry_id) -> None:
+async def async_remove_devices(
```

Comment:
Side note: This doesn't need to be a coroutine function as there's no await inside.